src/status_im/ui/screens/wallet/collectibles/cryptokitties.cljs

```diff
@@ -0,0 +1,90 @@
+(ns status-im.ui.screens.wallet.collectibles.cryptokitties
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :as re-frame]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.action-button.action-button :as action-button]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.screens.wallet.collectibles.views :as collectibles]
+            [status-im.utils.handlers :as handlers])
+  (:refer-clojure :exclude [symbol]))
+
+(def symbol :CK)
```

Comment:
do we really want to shadow standard func? It's a bit confusing imo

Reply:
Agree